src/pages/send/confirm/confirm.ts

```diff
@@ -645,6 +645,8 @@ export class ConfirmPage {
 
   private setSendError(msg: string) {
     if (this.isCordova) this.slideButton.isConfirmed(false);
+    if (msg === "FINGERPRINT_CANCELLED") return;
```

Comment:
This looks like it can work for android, but not for iOS because the error codes on iOS are different: https://ionicframework.com/docs/native/touch-id/

From the link above, here are the two possible "User Canceled" error codes`-2 or -128 - User tapped the 'Cancel' button`.

It might be worth consolidating the different iOS and Android messages in the `check` method of the touchid provider and then returning a consistent error message to all callers of the method when a users cancels touchid.